interpreter/exec/simd.ml

```diff
@@ -16,7 +28,9 @@ sig
   type bits
   val default : t (* FIXME good name for default value? *)
   val to_string : t -> string
+  val of_bits : bits -> t
   val to_bits : t -> bits
+  val of_strings : shape -> string list -> t
 end
 
 module Make (Rep : RepType) : S with type bits = Rep.t =
```

Comment:
Aside: Do we actually need this functor? It doesn't seem to do anything interesting.

Reply:
From https://github.com/WebAssembly/simd/pull/138#discussion_r345508572 it seemed like we wanted this for extensibility.

Reply:
Compatibility with some other code as well, there is some code that expects `RetType` and `S` distinction.